### PR TITLE
adding troubleshooting scenario

### PR DESCRIPTION
**Pull request by jooseppi-luna**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [x] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/troubleshooting/powerflex.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ description: Troubleshooting PowerFlex Driver
 | Symptoms | Prevention, Resolution or Workaround |
 |------------|--------------|
 | The installation fails with the following error message: <br />```Node xxx does not have the SDC installed```| Install the PowerFlex SDC on listed nodes. The SDC must be installed on all the nodes that need to pull an image of the driver. |
+| The standalone Helm chart installatino fails with `Error: couldn't find key MDM in Secret vxflexos/vxflexos-config` | Make sure that you have ssh keys set up between the master and the worker nodes. |
```
Comment (AronAtDell) [inline content/docs/csidriver/troubleshooting/powerflex.md:10]: "installatino" typo.

Comment (jooseppi-luna) [reply content/docs/csidriver/troubleshooting/powerflex.md:10]: duh, thanks

Comment (jooseppi-luna) [reply content/docs/csidriver/troubleshooting/powerflex.md]: @AronAtDell fixed

**Review by bharathsreekanth — APPROVED** [review]

**Review by mjsdell — APPROVED** [review]
Approved

**Pull request merged**